components/FeatureHighlight/src/FontThemer/MDCFeatureHighlightFontThemer.m

```diff
@@ -18,7 +18,7 @@
 
 @implementation MDCFeatureHighlightFontThemer
 
-+ (void)applyFontScheme:(id<MDCFontScheme>)fontScheme
```

Comment:
May be worth breaking the font themer updates out to a separate PR.

Reply:
Nah might be ok. Am always just mainly concerned about a PR getting stuck because of feedback on a part that’s not core to the PR.